ports/mimxrt/mpconfigport.h

```diff
@@ -58,15 +68,34 @@
 #define MICROPY_PY_BUILTINS_FILTER          (0)
 #define MICROPY_PY_BUILTINS_REVERSED        (1)
 #define MICROPY_PY_BUILTINS_MIN_MAX         (0)
+#define MICROPY_PY_BUILTINS_HELP            (1)
+#define MICROPY_PY_BUILTINS_HELP_MODULES    (1)
+#define MICROPY_PY_BUILTINS_HELP_TEXT       mimxrt_help_text
 #define MICROPY_PY___FILE__                 (0)
 #define MICROPY_PY_MICROPYTHON_MEM_INFO     (1)
 #define MICROPY_PY_ARRAY_SLICE_ASSIGN       (1)
 #define MICROPY_PY_COLLECTIONS              (0)
 #define MICROPY_PY_SYS_MAXSIZE              (1)
 
+// Fine control over Python builtins, classes, modules, etc
+#define MICROPY_PY_IO_IOBASE                (1)
+#define MICROPY_PY_IO_FILEIO                (1)
+#define MICROPY_PY_SYS_STDFILES             (1)
+#define MICROPY_PY_SYS_STDIO_BUFFER         (1)
+#define MICROPY_PY_SYS_PLATFORM             "i.mx rt106x"
```

Comment:
I think the platform should be the name of the port, ie `mimxrt`.

Reply:
AFAIK, I did that in the feature pack PR.

Reply:
I'm not confident about the state of this branch on my PC. So may we defer that to the mimxrt_features PR?

Reply:
I'll change it during merge of this PR.